src/ontology/cl-edit.owl

```diff
@@ -4849,12 +4849,16 @@ SubClassOf(obo:CL_0000164 obo:CL_0000163)
 # Class: obo:CL_0000165 (neuroendocrine cell)
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MESH:D055099"^^xsd:string) obo:IAO_0000115 obo:CL_0000165 "An endocrine cell that has the specialized function to produce and secrete hormones in response to neuronal signals."^^xsd:string)
+AnnotationAssertion(oboInOwl:hasAlternativeId obo:CL_0000165 "CL:0000381")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000165 "BTO:0002691"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000165 "FMA:83810"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000165 "neurosecretory cell"^^xsd:string)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000165 "neurosecretory neuron"^^xsd:string)
```

Comment:
Consider adding dbxref as noted in ticket.

Reply:
Added with the comment